runtime/lua/json.lua

```diff
@@ -0,0 +1,383 @@
+--
+-- json.lua
```

Comment:
Can you just use `vim.api.nvim_call_function('json_…'`? There already is a JSON parser in project, do not need to add a second one. If some features are missing they can be added.